core/lib/zksync_core/src/lib.rs

```diff
@@ -655,6 +660,8 @@ pub async fn initialize_components(
                 eth_sender.sender.clone(),
                 store_factory.create_store().await,
                 eth_client_blobs_addr.is_some(),
+                None,
```

Comment:
it should be coming from the config

Reply:
I think this might require more complex logic.. as after upgrade this should "autochange" to Some(Calldata)